fe/fe-core/src/main/java/com/starrocks/sql/parser/StarRocks.g4

```diff
@@ -686,6 +687,10 @@ killStatement
     : KILL (CONNECTION? | QUERY) INTEGER_VALUE
     ;
 
+killRunningQueryStatement
+    : KILL RUNNING QUERY string
+    ;
```

Comment:
There is already a statement `KILL QUERY conn_id`.
This is going to introduce a new statement, I think we need to discuss about its details first in the issue before we merge this in the code base.